armotypes/portaltypesutils.go

```diff
@@ -250,3 +251,12 @@ func (p *PortalBase) GetAttributes() map[string]interface{} {
 func (p *PortalBase) SetAttributes(attributes map[string]interface{}) {
 	p.Attributes = attributes
 }
+
+func (p *PortalBase) SetUpdatedTime(updatedTime string) {
+	_, err := time.Parse(time.RFC3339, updatedTime)
```

Comment:
It is a bit of a waste for the caller to make the time string and this function to make the string time again 
Also we want to control in one place the time format 
So maybe send *time.Time instead of string as argument and if it is nll then use current time